tests/run_tests.vsh

```diff
@@ -120,7 +121,8 @@ fn execute_c2v_command(options string, file string, c2v_dir string) {
 }
 
 fn try_get_generated_file(file string, test_file_extension string) !string {
-	generated_file := file.replace(test_file_extension, '.v')
+	mut re := regex.regex_opt('${test_file_extension}$') or { panic(err) }
+	generated_file := re.replace(file, '.v')
```

Comment:
Why do we need to use `regex` here and not `file.replace(test_file_extension, '.v')`?

Reply:
Same here but ".v.v.v.v.v.v.v"